py/desisim/templates.py

```diff
@@ -1812,6 +1817,10 @@ def make_templates(self, nmodel=100, zrange=(0.5, 4.0), rmagrange=(20.0, 22.5),
 
             if mag is None:
                 mag = rand.uniform(rmagrange[0], rmagrange[1], nmodel).astype('f4')
+	# Read in the BAL templates if needed. 
+        if balqso: 
+            baltemplatefile = os.environ['DESI_BASIS_TEMPLATES'] + "/../../trunk/BAL-templates-v0.2.fits"
+            balwave, baltemplates = bal.readbaltemplates(baltemplatefile)
 
```

Comment:
The location of the templates can't be hard-coded like this -- I will add them to $DESI_BASIS_TEMPLATES, desisim-testdata, and bump the version number.

Also, the templates should be read in *once* in `QSO.__init__` and cached.